test/data/enrichment/test_service.py

```diff
@@ -67,57 +67,99 @@ def assert_upload_dataframe(_, dataframe, schema, tablename, project, dataset):
     def test_enrichment_query_by_points(self):
         user_dataset = 'test_dataset'
         tablename = 'test_table'
+        geometry_column = 'the_geom'
         query_function = _prepare_sql_by_points
-        variables = pd.DataFrame([['table1.var1'], ['table1.var2']], columns=['id'])
+        variables = pd.DataFrame([['project.dataset.category_table1_comp_geog_geogyear_yearly_datayear.var1'],
```

Comment:
I see we are modifying the previous tests, but instead as @oleurud suggested we should add new tests instead:

- One for multiple variables from one dataset
- Another one for multiple variables from several datasets

Reply:
Yes we should. Based on this one, you can add both cases easily